Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -869,10 +869,13 @@ def genS(fragment,howMuch):
 steps['Cosmics_UP16']=merge([{'cfg':'UndergroundCosmicMu_cfi.py','-n':'500','--conditions':'auto:run2_mc_cosmics','--scenario':'cosmics','--era':'Run2_2016'},Kby(666,100000),step1Defaults])
 steps['Cosmics_UP17']=merge([{'cfg':'UndergroundCosmicMu_cfi.py','-n':'500','--conditions':'auto:phase1_2017_cosmics','--scenario':'cosmics','--era':'Run2_2017'},Kby(666,100000),step1Defaults])
 steps['Cosmics_UP18']=merge([{'cfg':'UndergroundCosmicMu_cfi.py','-n':'500','--conditions':'auto:phase1_2018_cosmics','--scenario':'cosmics','--era':'Run2_2018'},Kby(666,100000),step1Defaults])
+steps['Cosmics_UP19']=merge([{'cfg':'UndergroundCosmicMu_cfi.py','-n':'500','--conditions':'auto:phase1_2019_realistic','--scenario':'cosmics','--era':'Run3'},Kby(666,100000),step1Defaults])
 steps['CosmicsSPLoose_UP17']=merge([{'cfg':'UndergroundCosmicSPLooseMu_cfi.py','-n':'2000','--conditions':'auto:phase1_2017_cosmics','--scenario':'cosmics','--era':'Run2_2017'},Kby(5000,500000),step1Up2015Defaults])
 steps['CosmicsSPLoose_UP18']=merge([{'cfg':'UndergroundCosmicSPLooseMu_cfi.py','-n':'2000','--conditions':'auto:phase1_2018_cosmics','--scenario':'cosmics','--era':'Run2_2018'},Kby(5000,500000),step1Up2015Defaults])
 steps['BeamHalo']=merge([{'cfg':'BeamHalo_cfi.py','--scenario':'cosmics'},Kby(9,100),step1Defaults])
 steps['BeamHalo_13']=merge([{'cfg':'BeamHalo_13TeV_cfi.py','--scenario':'cosmics'},Kby(9,100),step1Up2015Defaults])
+steps['BeamHalo_UP18']=merge([{'cfg':'BeamHalo_13TeV_cfi.py','-n':'500','--conditions':'auto:phase1_2018_cosmics','--scenario':'cosmics','--era':'Run2_2018','--beamspot':'Realistic25ns13TeVEarly2018Collision'},Kby(666,100000),step1Defaults])
+steps['BeamHalo_UP19']=merge([{'cfg':'BeamHalo_13TeV_cfi.py','-n':'500','--conditions':'auto:phase1_2019_realistic','--scenario':'cosmics','--era':'Run3','--beamspot':'Realistic25ns13TeVEarly2017Collision'},Kby(666,100000),step1Defaults])
```

Comment:
@tocheng @tlampen @christopheralanwest 
please provide a comics Global Tag as requested here: https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/4124.html in order to move on with this PR. 
Thanks

Reply:
@mmusich 
For peak and deco mode each?

Reply:
@tocheng, for the time being deco mode is enough. Thanks

Reply:
@mmusich I am taking this chance to review whether any other tags in postLS2 queue need to be updated (to 2018 condition). Should be converged soon.

Reply:
@tocheng. OK. I suppose that it would be good to name the new queue something containing 2021 instead of 2019, in the spirit of https://github.com/cms-sw/cmssw/pull/26535 gets merged first.

Reply:
Hi @mmusich , in the same spirit can you introduce UP21 instead of UP19 in the new WFs ? Thanks

Reply:
Let's see how it goes and who gets merged first. The other can clean the remaining issues.

Reply:
well the decision to abandon 19 in profit of 21 is already taken , therefore one should from now on not introduce new '19 ' .

Reply:
@mmusich @boudoul was it changed to 21 or not? I don't see the changes

Reply:
@prebello it will be handled by a rebase of whichever PR does not get merged first

Reply:
In principle #26535 is supposed to deal with the migration, but this hasn't yet converged. Anyway I agree we should avoid doing and undoing things

Reply:
Well, this is fully signed and probably either one of us would need to rebase once the other gets merged. So somehow it is your call. I won't be able to edit this until next week, but since there is no criticality associated with this PR that is fine.